packages/shared/src/lib/core/network/constants/default-network-metadata.constant.ts

```diff
@@ -11,7 +11,6 @@ export const DEFAULT_NETWORK_METADATA: Readonly<{ [key in StardustNetworkId]?: N
     [SupportedNetworkId.Iota]: <IStardustNetworkMetadata>{
         id: SupportedNetworkId.Iota,
         name: 'IOTA',
-        networkName: StardustNetworkName.Iota,
```

Comment:
dont we need these anymore? we still set it in the constructor of `StardustClass`

Reply:
We get this information from the protocol (see `default-protocol.constant.ts`). For convenience reasons we set this inside `StardustClass` directly